content/en/real_user_monitoring/guide/mobile-sdk-upgrade.md

```diff
@@ -154,9 +172,138 @@ let package = Package(
 
 {{% /tab %}}
 
-{{< /tabs >}}
+{{% tab "React Native" %}}
+
+Update `@datadog/mobile-react-native` in your package.json:
+
+```json
+"@datadog/mobile-react-native": "2.0.0"
+```
+
+Update your iOS pods:
+
+```bash
+(cd ios && bundle exec pod install)
```

Comment:
should it be wrapped in parentheses?

Reply:
This is an alternative to `cd ios && bundle exec pod install && cd ..` that makes sure that if the `pod install` fails, we don't end up in the `ios` directory so we can re-run the command.
I realize that it should be `pod update` and not `pod install`, I'll change this.